entry.py

```diff
@@ -0,0 +1,7 @@
+from datetime import datetime
+
+
+class Entry:
+    def __init__(self, user_id: str):
+        self.timestamp = datetime.utcnow()
+        self.user_id = user_id
```

Comment:
models should be in a folder models/ because you will have way more than 1 in almost any project..

Reply:
i transfered the entry.py into the models directory

Reply:
i transfered user.py too